pkg/webhook/operatingsystemconfig/scripts/configure-containerd-registries.sh

```diff
@@ -10,13 +10,14 @@ function with_backoff {
     exit 1
   fi
 
-  local max_attempts=${MAX_ATTEMPTS-20}
```

Comment:
Having in mind that the func no longer does exponential backoff, you could rename the func to `retry` and add a detailed doc string about the custom retry strategy.